CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Next Release
 ### Features
-- [In Progress: 20th Jul 2020] Show assigned facility in patient summary screen
+- [In Progress: 21th Jul 2020] Patient assigned facility
```

Comment:
Is there something wrong with the CHANGELOG here?

We can have two different lines, right?

- Show assigned facility in the patient summary screen
- [In Progress] Use patient's assigned facility to schedule an appointment

Reply:
I feel like these changes all fall under patient assigned facility feature, should we track them separately?

Reply:
Yeah, I think we should. These are all small features that can go out separately.

Reply:
Done